pkgs/by-name/ne/neargye-semver/package.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "neargye-semver";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "Neargye";
+    repo = "semver";
+    tag = "v${finalAttrs.version}";
+    sha256 = "sha256-0HOp+xzo8xcCUUgtSh87N9DXP5P0odBaYXhcDzOiiXE=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+
+  buildInputs = [ ];
+
+  cmakeFlags = [ (lib.cmakeFeature "CMAKE_BUILD_TYPE" "Release") ];
+
```

```suggestion
  doCheck = true;
```
The package has a test suite we could run

---

Enabled!